### ISSUE TITLE
Breaking changes with 0.0.3!

### ISSUE THREAD

**Issue by bruvv** [rsnodgrass/hass-helium#15]
```
Invalid config for [sensor.helium]: expected a list for dictionary value @ data['helium_prefix']. Got None. (See /config/sensors.yaml, line 459).
```
After updating to 0.0.3 I get the above error. Nothing is mentioned in the changelog about this. Can you please add information regarding this ?

**Comment by rsnodgrass** [comment]
That slipped through, thanks.  Should be resolved with 0.3.1

**Comment by bruvv** [comment]
Thanks for the quick fix. Going from 0.0.3 to 0.3.1 seems a little crazy btw 👀
